### DISCUSSION TITLE
Input group with addon (prepend/append) in the middle

### DISCUSSION THREAD

**Discussion by septatrix** [twbs/bootstrap#32908] [Q&A] [answered]
This was easy in BS3. I am now converting an old project to BS4 (and soon BS5) and wanted to keep this this layout in a form. After a bit of searching I stumbled upon this answer and would like to know if this is intended/officially supported: https://stackoverflow.com/questions/53593500/input-group-with-a-prepended-element-in-the-middle

**Comment by mdo** [comment, marked as answer]
This is coming in v5: https://getbootstrap.com/docs/5.0/forms/input-group/#basic-example.

**Reply by septatrix** [reply]
Okay, so I will use the workaround for v4 and drop it when upgrading. Thanks

**Comment by satya013** [comment]
View in github
